src/Sulu/Bundle/MediaBundle/Media/Manager/MediaManagerInterface.php

```diff
@@ -19,6 +19,7 @@
 
 /**
  * @method string getAdminUrl($id, $fileName, $version);
+ * @method void removeFileVersion(int $mediaId, int $version);
```

Comment:
okay with this the check above actually makes sense. but as we are releasing a minor version, i would just add the method to the interface. should not be hard to implement this in a project and will save us a bit of pain in the future 🙂